daemon/src/oracle.rs

```diff
@@ -388,3 +380,15 @@ impl Attestation {
 impl xtra::Message for Attestation {
     type Result = ();
 }
+
+#[cfg(test)]
+pub mod tests {
+    #[test]
+    fn ensure_lookahead_constant() {
+        use time::Duration;
+        assert_eq!(
+            crate::oracle::ANNOUNCEMENT_LOOKAHEAD,
+            model::SETTLEMENT_INTERVAL + Duration::hours(2)
```

Comment:
maybe we could write down the reasoning for +2 here or at the constant definition?